packages/schemas/src/multiVariableText/propPanel.ts

```diff
@@ -15,6 +15,7 @@ const mapDynamicVariables = (props: PropPanelWidgetProps) => {
     changeSchemas([
       { key: 'content', value: JSON.stringify(variables), schemaId: activeSchema.id },
       { key: 'variables', value: varNames, schemaId: activeSchema.id },
+      { key: 'readOnly', value: varNames.length === 0, schemaId: activeSchema.id },
```

Comment:
setting readOnly here, at the same time as the variables and content works around the #1009 problem